realm/realm-library/src/androidTest/assets/ios/README.md

```diff
@@ -1,11 +1,33 @@
 This folder contains various Realm databases created on iOS and can be used to test interop with
-Realm-Android.
+Realm-Android. The databases are generated using the below iOS code.
 
-The databases are generated using the following iOS code:
+(02/10-2015) Note that debugging must be disabled as it is currently not possible to debug encrypted
```

Comment:
Uh, I think we don't have this limitation anymore after remove signal handler for encryption? Worth to check with cocoa team.